inc/vcf/fixer.hpp

```diff
@@ -156,11 +157,68 @@ namespace ebi
             util::writeline(output, *line);
             ignored_errors++;
         }
+
+        /**
+         * explanation of the fix:
+         * - in error.get_field there will be the field name as it appears in the FORMAT column, e.g. "GT".
+         * - don't do anything if error.get_field() is empty
+         * - get the index in the FORMAT column of error.get_field(), e.g. in GT:AC:AN, the index of AC is 1.
+         * - for each sample column
+         *      - put a "." instead of the value in the index-th position of ":" e.g. 0/0:3:4 becomes 0/0:.:4 for index = 1
+         * @param error
+         */
         virtual void visit(SamplesBodyError &error) override
         {
-            util::writeline(output, *line);
-            ignored_errors++;
+            if (error.get_field().empty()) {
```

Comment:
I did not make it mandatory because the next errors are not related to any specific field:
- "the format lists a different number of fields than the sample column has"
- "The number of samples must match those listed in the header line"
- general syntax errors detected in ragel